sjb/config/test_cases/test_branch_origin_aggregated_logging_json_file.yml

```diff
@@ -0,0 +1,10 @@
+---
+parent: 'test_cases/test_branch_openshift_ansible_logging_json_file.yml'
+overrides:
+  sync:
+    - "openshift,origin=master"
+    - "openshift,aos-cd-jobs=master"
+    - "openshift,image-registry=master"
+    - "openshift,openshift-ansible=master"
```

Comment:
This is an O-A-L job so we need to specify we want O-A, whereas the parent specifies O-A-L here. Whatever code is under test gets synced by default. This is just for other dependencies.